packages/lesspass-pure/src/views/Login.vue

```diff
@@ -107,6 +113,53 @@ export default {
         this.$store.dispatch("setBaseURL", { baseURL });
         User.login({ email: this.email, password: this.password })
           .then(response => {
+            User.getLoggedUserInformation().then(response => {
+              if (response.data.key === null) {
+                LessPassEntropy.generateUserKey().then(key => {
+                  const encryptedKey = LessPassCrypto.encrypt(
+                    key,
+                    this.password
+                  );
+                  Password.all().then(passwords => {
```

Comment:
This part is potentially long. I'm wondering if we need to move the encryption page on a dedicated page. Display a loading icon during the operation. And handle the encryption failure of one password with some retry